README.md

```diff
@@ -65,24 +66,45 @@ Create a file named `kroki.html` with the following content and open it in your
       var kroki = AsciidoctorKroki
 
       const registry = asciidoctor.Extensions.create()
-      kroki.register(registry)
+      kroki.register(registry) // <1>
       var result = asciidoctor.convert(input, {'extension_registry': registry})
       document.getElementById('content').innerHTML = result
     </script>
   </body>
 </html>
 ```
-**<1>** Register the extension in the global registry <br/>
-**<2>** Register the extension in a dedicated registry
+**<1>** Register the extension in a dedicated registry
 
 ## Usage
 
-```adoc
-[plantuml,alice-bob,svg,role=sequence]
-....
-alice -> bob
-....
+Kroki currently supports the following diagram libraries:
+
+* ActDiag: `actdiag`
```

Comment:
It's probably a bit of effort but as an end user who is not familiar with all of these I would like each one to be a hyperlink to the library location.